### ISSUE TITLE
Multiplayer bug

### ISSUE THREAD

**Issue by Furious-mat** [D00Med/vehicles#65]
Hi, I don't know if this is normal but I've been playing multiplayer for Lego city undercover Minetest (a game I'm currently doing, by the way a small army of new vehicles would be cool) but I've noticed that in multiplayer when 1 person is driving in a vehicle and a 2nd person is driving the vehicles become very slow, (basically I wanted to race but we were very slow when we were driving at the same time) is that normal?

**Comment by D00Med** [comment]
Yes, it's normal. The vehicles aren't well optimised since a lot of functionality got tacked on over the years.
